src/MLJModelInterface.jl

```diff
@@ -80,7 +80,7 @@ end
 export UnivariateFinite
 
 # parameter_inspection:
-export params
+export params, flat_params
```

Comment:
Please do not export `flat_params`. In tests you can call it as `MLJModelInterface.flat_params`.